src/utils/viewer/viewer.cpp

```diff
@@ -90,14 +87,55 @@ class Viewer : public Mn::Platform::Application {
   void torqueLastObject();
   void removeLastObject();
   void wiggleLastObject();
-
-  void recomputeNavMesh(const std::string& sceneFilename,
-                        esp::nav::NavMeshSettings& navMeshSettings);
-
   void invertGravity();
   Mn::Vector3 randomDirection();
 
-  void toggleNavMeshVisualization();
+  std::string helpText = R"(
+==================================================
+Welcome to the Habitat-sim C++ Viewer application!
+==================================================
+Mouse Functions:
+----------------
+  LEFT:
+    Click and drag to rotate the agent and look up/down.
+  RIGHT:
+    (With 'enable-physics') Click a surface to instance a random primitive object at that location.
+  SHIFT-RIGHT:
```

Comment:
@aclegg3 : FYI. The object picking function does not work any more.

Reply:
It is working for me 🤔